releases/8.0/ja.php

```diff
@@ -0,0 +1,508 @@
+
+  <div class="php8-columns">
+    <div class="php8-column">
+      <h2 class="php8-h2 php8-h2_margin-top">型システムとエラーハンドリングの改善</h2>
+      <ul>
+        <li>
+          算術/ビット演算子のより厳密な型チェック
+          <a href="https://wiki.php.net/rfc/arithmetic_operator_type_checks">RFC</a>
+        </li>
+        <li>
+          トレイトの抽象メソッドの検証 <a href="https://wiki.php.net/rfc/abstract_trait_method_validation">RFC</a>
+        </li>
+        <li>
+          マジックメソッドのシグナチャ <a href="https://wiki.php.net/rfc/magic-methods-signature">RFC</a>
+        </li>
+        <li>
+          エンジンの警告を整理 <a href="https://wiki.php.net/rfc/engine_warnings">RFC</a>
+        </li>
+        <li>
+          互換性のないメソッドシグナチャは fatal error に。
+          <a href="https://wiki.php.net/rfc/lsp_errors">RFC</a>
+        </li>
+        <li>
+          @ 演算子は、致命的なエラーを隠さなくなりました。
+        </li>
+        <li>
+          private メソッドの継承時のシグナチャチェック <a href="https://wiki.php.net/rfc/inheritance_private_methods">RFC</a>
+        </li>
+        <li>
+          Mixed 型のサポート <a href="https://wiki.php.net/rfc/mixed_type_v2">RFC</a>
+        </li>
+        <li>
+          戻り値で static 型をサポート <a href="https://wiki.php.net/rfc/static_return_type">RFC</a>
+        </li>
+        <li>
+          内部関数に型アノテーション
+          <a href="https://externals.io/message/106522">Email thread</a>
+        </li>
+        <li>
+          一部の拡張機能が、リソースからオブジェクトに移行:
+            <a href="https://php.watch/versions/8.0/resource-CurlHandle">Curl</a>,
+            <a href="https://php.watch/versions/8.0/gdimage">Gd</a>,
+            <a href="https://php.watch/versions/8.0/sockets-sockets-addressinfo">Sockets</a>,
+            <a href="https://php.watch/versions/8.0/OpenSSL-resource">OpenSSL</a>,
+            <a href="https://php.watch/versions/8.0/xmlwriter-resource">XMLWriter</a>,
+            <a href="https://php.watch/versions/8.0/xmlwriter-resource">XML</a>
+        </li>
+      </ul>
+    </div>
+    <div class="php8-column">
+      <h2 class="php8-h2 php8-h2_margin-top">その他文法の調整や改善</h2>
+      <ul>
+        <li>
+          引数やクロージャーのuseリストの最後にカンマがつけられるように <a href="https://wiki.php.net/rfc/trailing_comma_in_parameter_list">RFC</a>
+          <a href="https://wiki.php.net/rfc/trailing_comma_in_closure_use_list">RFC</a>
+        </li>
+        <li>
+          catch で例外のキャプチャが不要に <a href="https://wiki.php.net/rfc/non-capturing_catches">RFC</a>
+        </li>
+        <li>
+          変数の文法の調整 <a href="https://wiki.php.net/rfc/variable_syntax_tweaks">RFC</a>
+        </li>
+        <li>
+          名前空間の名前を単一のトークンとして扱う <a href="https://wiki.php.net/rfc/namespaced_names_as_token">RFC</a>
+        </li>
+        <li>
+          Throw は式になりました <a href="https://wiki.php.net/rfc/throw_expression">RFC</a>
+        </li>
+        <li>
+          オブジェクトに対して ::class が使えるように <a href="https://wiki.php.net/rfc/class_name_literal_on_object">RFC</a>
+        </li>
+      </ul>
+
+      <h2 class="php8-h2 php8-h2_margin-top">新しいクラス、インターフェイス、関数</h2>
```

Comment:
I'm far from an expert of Japanese, but as the list is exhaustive maybe the following?
```suggestion
      <h2 class="php8-h2 php8-h2_margin-top">新しいクラスとインターフェイスと関数</h2>
```

Reply:
@Girgias 

「新しいクラス、インターフェイス、関数」is natural expression for me.

(形容詞/adjective) "新しい" does adjective "クラス、インターフェイス、関数", like as original English title.

Although unlikely, 「新しいクラスとインターフェイスと関数」makes  an impression like 
「 "新しいクラス" と  "インターフェイスと関数" 」, in other words
"新しいクラスと（それにともなう）インターフェイスと関数（の関係）"  ~ relationship between new class and "interface and function"

Reply:
Yes, I split them by "、" because they are independent of each other.